Integrations/Whois/Whois.py

```diff
@@ -8361,11 +8360,11 @@ def whois_command():
 
 
 def test_command():
-    whois_result = get_whois('google.com')
+    whois_result = get_whois('google.co.uk')
 
-    domain_test = whois_result['id'][0]
+    domain_test = whois_result['nameservers'][0]
```

Comment:
change your code to be more protective. otherwise the user will get not indicative error like `out of index`